features/steps/api_with_persona_steps.py

```diff
@@ -336,6 +339,103 @@ def create_provider_machine(context):
     context.persona['provider_machine'] = provider_machine
 
 
+@step('we get the projects via the API')
+def create_project_api(context):
+    assert context.persona
+    client = context.persona['client']
+    url = '/api/v2/projects'
+    response = client.get(url)
+    context.persona['response'] = response
+
+
+@step('we create a project called "{project_name}" via the API')
+def create_project_api(context, project_name):
+    assert context.persona
+    client = context.persona['client']
+    user = context.persona['user']
+    from core.models import AtmosphereUser, Identity
+    assert isinstance(user, AtmosphereUser)
+    # user_identity = context.persona['user_identity']
+    # assert isinstance(user_identity, Identity)
+    # groups = user.groups.all()
+    # context.test.assertEqual(len(groups), 1)  # If this is not True then find some other way to find the group.
+    # owner_group = groups[0]
+    url = '/api/v2/projects'
+    response = client.post(url,
+                           {
+                               'name': project_name,
+                               'description': project_name,
+                               'owner': user.username
+                           })
+    context.persona['response'] = response
+
+
+@when('I create a volume with name "{volume_name}" and size {volume_size:d} using API')
```

Comment:
@lenards I made the wording more consistent while squashing and tidying things up. Ta!